herddb-core/src/test/java/herddb/file/FileCommitLogTest.java

```diff
@@ -180,59 +184,63 @@ public void accept(LogSequenceNumber t, LogEntry u) {
 
     @Test
     public void testLogsynch() throws Exception {
-        FileCommitLogManager manager = new FileCommitLogManager(folder.newFolder().toPath(), 64 * 1024 * 1024);
-        int writeCount = 0;
-        final long _startWrite = System.currentTimeMillis();
-        try (CommitLog log = manager.createCommitLog("tt", "aa", "nodeid");) {
-            log.startWriting();
-            for (int i = 0; i < 100; i++) {
-                log.log(LogEntryFactory.beginTransaction(0), true);
-                writeCount++;
-            }
-        }
-        final long _endWrite = System.currentTimeMillis();
-        AtomicInteger readCount = new AtomicInteger();
-        try (CommitLog log = manager.createCommitLog("tt", "aa", "nodeid");) {
-            log.recovery(LogSequenceNumber.START_OF_TIME, new BiConsumer<LogSequenceNumber, LogEntry>() {
-                @Override
-                public void accept(LogSequenceNumber t, LogEntry u) {
-                    readCount.incrementAndGet();
+        try (FileCommitLogManager manager = new FileCommitLogManager(folder.newFolder().toPath(), 64 * 1024 * 1024);) {
```

Comment:
this change seems big, but it is only about wrapping new FileCommitLogManager in a try with resources